src/components/new-request/business-lookup.vue

```diff
@@ -77,17 +80,43 @@ export default class BusinessLookup extends Vue {
   /** State of this component. */
   state = States.INITIAL
 
+  /** Business Lookup labels and warnings. */
+  lookupLabel = ''
+  lookupNoActiveText = ''
+
+  /** Set business lookup text on mount. */
+  mounted (): void {
+    this.setLookupText()
+  }
+
   /** Called when searchField property has changed. */
   @Watch('searchField')
   onSearchFieldChanged (): void {
     this.onSearchInputDebounced(this)
   }
 
+  /** Called when business lookup search status prop has changed. */
+  @Watch('businessStatus')
+  onStatuschanged (): void {
+    this.setLookupText()
+  }
+
+  /** Business lookup text based on the business status to search for. */
+  private setLookupText (): void {
```

Comment:
Felt like this is the cleanest way to do it instead of having `v-ifs` in the template.

Reply:
If this ends up being called from only 1 place, I recommend moving this code there.

Reply:
I'm calling it twice Sev. I'm calling it on mount and when the status is changed (for example moving from Name change to Restoration). Both of them use the business lookup.

Reply:
😁

Reply:
Looks good.